src/kernels/jupyter/types.ts

```diff
@@ -234,13 +234,21 @@ export interface IJupyterUriProviderRegistration {
     getJupyterServerUri(id: string, handle: JupyterServerUriHandle): Promise<IJupyterServerUri>;
 }
 
+export interface IJupyterServerUriEntry {
+    uri: string;
+    serverId: string;
+    time: number;
```

Comment:
Some comment here would be good, is this time this was added or last accessed?

Reply:
Good call. I'll add that in and update.